lerobot/scripts/push_dataset_to_hub.py

```diff
@@ -40,60 +40,6 @@
 --raw-format umi_zarr \
 --repo-id lerobot/umi_cup_in_the_wild
 ```
-
-**WARNING: Updating an existing dataset**
```

Comment:
FYI @alexander-soare , we decided to move it in another markdown. This is because common users dont care about this info. It is mainly for maintainers.